contributor/04-mechanics-of-contributing.md

```diff
@@ -0,0 +1,160 @@
+You'll have to understand how the host team would like you to reach out for this.
+It's best to ask a trusted committer about how to best discuss your proposal.
+
+It is time-and-again-proven wisdom from the Open Source arena that, if you get to select how to discuss your proposal you should try to select a written way.
+Ideally one whose artifacts are public, searchable and perma-linkable to enable referencing your proposal in later discussions on this future contribution or other contributions to come - by you or others. 
+
+This type of high-level, up-front agreement early will save time in rework or rejection of your pull request down the road.
+
+## Creating the pull request
+
+### Communication and unblocking yourself
+
+Great, you've made yourself familiar with the host team's approach and they are looking forward to receive your pull request.
+Which gotchas are there waiting for you now?
+
+First, you'll be in less direct contact with them and second you aren't expected to be as knowledgeable and proficient as you might be on the full-time projects that your team owns.
+How can you now deal with this the best?
+
+Try to peruse their documentation, the conversation archives and code artifacts from the host team to unblock yourself.
+This is similar to the situation you and likely most people find yourself in when using one of the popular OSS projects.
+
+Much like in Open Source projects, ask the host team if things are going nowhere even after trying to unblock yourself.
+The questions you ask and the answers you receive will help others coming after you solve the same issues.
+Make sure that your communication ends up in a searchable archive that is closely linked to the project itself.
+It doesn't do either side any good for you to spin forever on a problem that could be resolved in a few-minute conversation with someone more knowledgeable about the project.
+It's OK to ask for help.
+
+There's one key difference though, bringing advantage to you and other people in the future: 
+In almost all cases you should prefer the projects' official communication channels - this can be a mailing list, a chat room, the issue tracker or something similar depending on the purpose of having a more synchronous or asynchronous way of interacting or the varying needs for structure in the communication.
+All of those usually have in common that they are text-based, archived, searchable and come with stable links - this means your question and the answer will be written down, references you link in those answers will also be kept reachable.
+This way you could benefit from this passively documented knowledge on your search and will help future contributors to have the same advantage.
+Imagine what would be needed for realizing this benefit by just walking by or calling a trusted committer on the project or even in a group call.    
```

Comment:
Removing this line as suggested 
 by @rrrutledge  [here](https://github.com/InnerSourceCommons/InnerSourceLearningPath/pull/117/files#r324852819)

```suggestion
```

Reply:
👍 Thank you!

Reply:
Please commit the suggestion?